lib/create-space-api.js

```diff
@@ -173,325 +74,72 @@ export default function createSpaceApi ({
       .then((response) => wrapSpace(http, response.data), errorHandler)
   }
 
-  /**
```

Comment:
All these JSDoc comments are moved to typings files. When we refactor this file to TS the comments should be moved back to be collocated with an actual implementation again.